### DISCUSSION TITLE
Local audio player

### DISCUSSION THREAD

**Discussion by roelwelters** [music-assistant/support#3283] [Q&A] [answered]
I run HAOS on a mac mini and would like to connect the audio out of the mini to some speakers and play music with music assistant on them. It seems very obvious but I can’t figure out how to set the mini as a player.

**Comment by OzGav** [comment, marked as answer]
https://music-assistant.io/faq/stream-to/

**Comment by OzGav** [comment]
Local audio output has been added to 2.9.0beta4
